lzw.py

```diff
@@ -27,17 +26,39 @@
 from predictive import PredictiveTransform
 
 class LZWCodec:
+    """
+    This codec makes a predictive transformation before encoding. LZW is used to encode and decode afterwards.
+
+    Use only encode() and decode()
+    """ 
+
     def __init__(self):
+        """
+        Codec constructor.
+        """
         pass
-    
+
+
     def encode_LZW(self,data):
+        """
+        Encodes a given data using LZW.
+
+        Parameters:
+            data (one-dimensional array): the data to be encoded
+
+        Returns:
+            compressed_data: LZW encoded data
```

Comment:
Comments on returns must be in this format:

Returns:
     string: LZW encoded data

ONLY THE DATATYPE!